install/kubernetes/cilium/values.yaml

```diff
@@ -135,13 +135,6 @@ extraHostPathMounts: []
   #   readOnly: true
   #   mountPropagation: HostToContainer
 
-# -- Additional agent ConfigMap mounts.
-extraConfigmapMounts: []
```

Comment:
Can you please also remove `extraConfigmapMounts` from `Documentation/spelling_wordlist.txt`?